src/features/jobTracker/ui/JobEditor/JobEditor.tsx

```diff
@@ -26,14 +26,14 @@ export function JobEditor({ jobId, isNew, values, ...rest }: CardEditorProps) {
 
   return (
     <ModalOverlay
-      className="fixed inset-0 bg-black/50 overflow-y-auto z-200"
+      className="fixed inset-0 bg-black/60 overflow-y-auto z-200"
```

Comment:
is this legit syntax or a typo?

"bg-black/60"

Reply:
color/# syntax is tailwind opacity for rgba(), I guess so I darkened the background to remove background distractions.